yellowpages.lic

```diff
@@ -0,0 +1,56 @@
+#Load base contacts YAML
+Contacts = YAML.load_file('./scripts/data/base-contacts.yaml')
+Contacts = Contacts['contacts']
+
+#DEBUG INFO
+#puts "---Base Contacts---"
+#puts Contacts.inspect
+#puts Contacts.class
+
+#Check for <CharName>-contacts.yaml file and merges it if exists
+if File.exist?('./scripts/profiles/'+checkname+'-contacts.yaml')
+	puts "Personal Contact file found! Now importing!"
+	
+	Personal_Contacts = YAML.load_file('./scripts/profiles/'+checkname+'-contacts.yaml')
+	Personal_Contacts = Personal_Contacts['contacts']
+
+	#DEBUG INFO
+	#puts "---Personal Contacts---"
+	#puts Personal_Contacts.inspect
+	#puts Personal_Contacts.class
+
+	Contacts.merge!(Personal_Contacts)
+end
+
+#DEBUG INFO
+#puts "---Merged Contacts---"
+#puts Contacts.inspect
+#puts Contacts.class
+#puts "-----"
+
+
+arg_definitions = [
+    [
+    { name: 'search', regex: /[a-zA-Z]/, optional: false, description: 'Name of player to look up in contacts.' }

```

Comment:
I recommend having a `debug` arg that will output your debug messages (instead of leaving them commented out).